draft-duke-quic-version-aliasing.md

```diff
@@ -260,6 +268,26 @@ incoming version number and ITE. It MUST NOT use client controlled information
 other than the version number and ITE; for example, the client's IP address and
 port.
 
+## Packet Type Generation
+
+The server generates the packet type codepoint for each of the four long header
+packet types (Initial, 0RTT, Handshake, and Retry). Each of these codepoints is
+two bits.
+
+Future versions of QUIC with 4 or fewer long header packet types can specify a
+mapping of these fields to their types.
+
+Note that the server needs to derive the type codepoints solely from the version
+number. It cannot extract the token, and the token extension, until the packet
+is identified as an Initial packet.
+
+A straightforward implementation might take arbitrary bits from a hash of the
+version number. The first two bits it reads are the codepoint for Initial
+packets. The next pair of bits that is not a duplicate of the first is the
+codepoint for 0RTT packets. The next pair that does not duplicate the first two
+is the codepoint for Handshake packets, and the remaining codepoint is the
+Retry packet.
```

Comment:
"the remaining codepoint" isn't the same as the others, nor do I think it works.  "and so on" might suffice here.

Reply:
I'm not sure what you mean. Once you've found three of the codepoints from your entropy source, there's one left that by definition isn't the same.

I agree that this example could just say "and so on".  IMO it's clearer to expend a sentence to make it crystal clear how this might work. But obviously it's unclear as I've written it, so I'll look at rewording.